homeassistant/components/flexit_bacnet/entity.py

```diff
@@ -0,0 +1,25 @@
+"""Base entity for the Flexit Nordic (BACnet) integration."""
+from __future__ import annotations
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import FlexitCoordinator
+
+
+class FlexitEntity(CoordinatorEntity[FlexitCoordinator]):
+    """Defines a Flexit entity."""
+
+    def __init__(self, coordinator: FlexitCoordinator) -> None:
```

Comment:
Please move the `_attr_has_entity_name = True` to the base entity, you will need this in the other platforms as well